provisioning/ansible/roles/beetbox-ubuntu/tasks/main.yml

```diff
@@ -47,7 +38,3 @@
   with_items:
     - { regexp: '^UseDNS', line: 'UseDNS no' }
     - { regexp: '^GSSAPIAuthentication', line: 'GSSAPIAuthentication no' }
-
```

Comment:
@alexdesignworks this was moved to the `beetbox-virtualbox` role as it is required for the install of VB guest additions.